### PR TITLE
 CM_TMP_CURRENT_SCRIPT_PATH made accessible while updating dynamic env from main script

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: automation/script/module.py [hunk 1]
```diff
@@ -744,6 +744,8 @@ def _run(self, i):
         meta = script_artifact.meta
         path = script_artifact.path
 
+        env['CM_TMP_CURRENT_SCRIPT_PATH'] = path
```
Comment (arjunsuresh) [inline automation/script/module.py:747]: Hi @anandhu-eng can we make this assignment via a function `_update_env` where after the assignment we also call `update_env_with_values ` to update any dynamic values?

Comment (anandhu-eng) [reply automation/script/module.py:747]: Sure @arjunsuresh

File: automation/script/module.py [hunk 2]
```diff
@@ -4359,6 +4360,20 @@ def any_enable_or_skip_script(meta, env):
 
     return False
 
+############################################################################################################
+def _update_env(env, key=None, value=None):
+    if key == None or value == None:
+        return {'return': 1, 'error': 'None value not expected in key and value arguments in _update_env.'}
+    if not isinstance(key, str):
```
Comment (arjunsuresh) [inline automation/script/module.py:4367]: I think we should avoid this str check. We may even have list here.

Comment (anandhu-eng) [reply automation/script/module.py:4390]: Would that be for updating the dynamic env variables through `update_env_with_values`?

Comment (anandhu-eng) [reply automation/script/module.py:4390]: resolved over call

File: automation/script/module.py [hunk 3]
```diff
@@ -387,9 +388,9 @@ def _run(self, i):
         elif 'v' in i: verbose=i['v']
         
         if verbose:
-           env['CM_VERBOSE']='yes'
-           run_state['tmp_verbose']=True
-           logging.getLogger().setLevel(logging.DEBUG)
+            env['CM_VERBOSE']='yes'
```
Comment (arjunsuresh) [inline automation/script/module.py:391]: extra space?

Comment (anandhu-eng) [reply automation/script/module.py:391]: Hi @arjunsuresh , would this be a problem as most of the standard editors are preset with 4 spaces for a Tab. Previously, there was only 3. It is not matching with indendation of [previous](https://github.com/GATEOverflow/cm4mlops/blob/4b6000c7261e7e0ee5172956e58650f9b168551c/automation/script/module.py#L384) if.

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**